packages/terra-status/src/terra-dev-site/doc/example/colors.css

```diff
@@ -0,0 +1,3 @@
+.high-status {
```

Comment:
```suggestion
.attention-status {
```

Reply:
Fixed in https://github.com/cerner/terra-core/pull/2211/commits/618ae6178edc4a734ef31a044c3bc8d570051b5c